app/server/src/main/scala/org/bitcoins/server/BitcoindRpcBackendUtil.scala

```diff
@@ -383,51 +386,59 @@ object BitcoindRpcBackendUtil extends Logging {
         txids
       }
 
-    val processing = new AtomicBoolean(false)
-
-    system.scheduler.scheduleWithFixedDelay(0.seconds, interval) { () =>
-      {
-        if (processing.compareAndSet(false, true)) {
-          logger.debug("Polling bitcoind for mempool")
-          val numParallelism = {
-            val processors = Runtime.getRuntime.availableProcessors()
-            //max open requests is 32 in akka, so 1/8 of possible requests
-            //can be used to query the mempool, else just limit it be number of processors
-            //see: https://github.com/bitcoin-s/bitcoin-s/issues/4252
-            Math.min(4, processors)
-          }
+    for {
+      rescanning <- wallet.isRescanning()
+    } yield {
+      if (!rescanning) {
```

Comment:
This means if we start the wallet and we are rescanning, we will never start polling bitcoind after the rescan is finished.